pkgs/tools/misc/vimv/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, bash, fetchFromGitHub }:
+stdenv.mkDerivation {
+  pname = "vimv";
```

Comment:
uhm, there should be a version :D

Reply:
if there isn't one upstream, should i use the date of the last commit or the beginning of the commit hash?

Reply:
example: `version = "unstable-2019-23-24"`
date is date of commit